.github/workflows/reviewdog.yml

```diff
@@ -10,7 +10,7 @@ jobs:
   diff-review:
     runs-on: ubuntu-latest
     container:
-      image: golang:1.20
+      image: golang:1.20@sha256:cfc9d1b07b1ef4f7a4571f0b60a99646a92ef76adb7d9943f4cb7b606c6554e2
```

Comment:
i don't think we want to use golang container and set it up like the other repos.